modules/nodes-pods-autoscaling-creating-memory.adoc

```diff
@@ -6,9 +6,16 @@
 
 = Creating a horizontal pod autoscaler object for memory utilization by using the CLI
```

Comment:
```suggestion
[id="nodes-pods-autoscaling-creating-memory_{context}"]
= Creating a horizontal pod autoscaler object for memory utilization by using the CLI
```

Reply:
The extra space between the anchor ID might continue to work, but I suggest removing the extra space while you are in this file.